OracleDatabase/SingleInstance/samples/prebuiltdb/README.md

```diff
@@ -1,56 +1,33 @@
 # Example of creating an image with pre-built DB
-***Warning: The description below requires changes in the dockerfile related to the version and edition in question (docker-images/OracleDatabase/SingleInstance/dockerfiles/$VERSION/Dockerfile). It's recommended that you revert the changes after you have completed creating image without VOLUME (see next section).***
 
-## 1. Create an image without a VOLUME
+## 1. Startup a container and create the database
 
-When creating an image with a pre-build DB, we first need an image without a VOLUME. To create such an image you'll need to comment out the VOLUME command in the dockerfile in use (docker-images/OracleDatabase/SingleInstance/dockerfiles/$VERSION/Dockerfile).
+First, you need to start up a container to get a database created. For this you will have to have the image already built.
```

Comment:
Now that the `Dockerfiles` don't have `VOLUME` defined, can't this just be a `Dockerfile` itself, i.e. can't we provide an image that uses `FROM oracle/database:XX` with a script that just creates the database and primes it to start on container launch from this image? 

The steps as documented below are mostly an anti-pattern and without the `VOLUME` restriction, shouldn't be necessary. It would require splitting the current `CMD` script in two (DB creation and DB startup).

Reply:
Please don't mind that example, it will soon be obsolete. I'm going to enhance the build script to produce a prebuilt database image or the one we have right now, as per our earlier discussions.

Reply:
Perhaps remove the entire example then?

Reply:
I will, once the build script is updated. Right now this is just changed to the example that is consistent with the rest of the changes in this PR. Removing it now would just introduce inconsistency and discomfort for users who are still looking for this sample.